staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -116,8 +117,11 @@ type Request struct {
 	subresource  string
 
 	// output
-	err  error
-	body io.Reader
+	err error
+
+	// only one of body / bodyBytes may be set. requests using body are not retriable.
```

Comment:
We could make them retriable by using e.g. a TeeReader to store what we read and send.

(I don't think we should do this but it's not impossible)

Reply:
yup, I replied to that suggestion at https://github.com/kubernetes/kubernetes/pull/113933#issuecomment-1317131329

Reply:
If you want to stop getting this comment, you could make the comment say something like "this could be fixed with TeeReader but we don't have any evidence it's needed" :)